PF1/Assignment1/OctalConverter.java

```diff
@@ -0,0 +1,56 @@
+package assignment1;
+// 
+public class OctalConverter {
+	
+		int binaryToOctal(int binum)
+		{
+			int temp=0;
+			try
+			{int no=binum;
+				
+				while(no>0){
+				if(no%10!=1 | no%10!=0)
```

Comment:
if condition will always be true. e.g. no=100, So in this in the first iteration it will get true and will throw an exception every time.